src/uct/ib/configure.m4

```diff
@@ -431,6 +431,8 @@ AS_IF([test "x$with_ib" = "xyes"],
         with_mlx5_dv=no
     ])
 
+AS_IF([test "x$with_mlx5_dv" = "xyes"], [AC_DEFINE([HAVE_MLX5_DV], 1, [MLX5 DV support])])
```

Comment:
why need it outside of `with_ib`? Also it seems to be already checked above

Reply:
> why need it outside of `with_ib`? Also it seems to be already checked above

better code reading. fixed